server/cp/set_byline_on_publish.py

```diff
@@ -14,6 +25,10 @@ def set_byline_on_publish(sender, item, updates, **kwargs):
         return
 
     if not updated.get("authors"):
+        source = updated.get("source", "").strip().lower()
+        if source and source not in CP_SOURCES:
+            logger.warning(f"Skipping default author for non-CP source: {source}")
```

Comment:
it will be a lot of cases with the amount of items they are ingesting, so I wouldn't log it at all or make it a debug level